README.md

```diff
@@ -582,6 +583,27 @@ async.some(['file1','file2','file3'], fs.exists, function(result){
 
 ---------------------------------------
 
+<a name="someLimit" />
+### someLimit(arr, limit iterator, callback)
+
+__Alias:__ `anyLimit`
```

Comment:
I might be mistaken but it appears the alias is stated here but remains un-wired in the code?